stem/model.py

```diff
@@ -164,6 +164,10 @@ def add_load_by_coordinates(self, coordinates: Sequence[Sequence[float]], load_p
         if isinstance(load_parameters, MovingLoad):
             self.__validate_moving_load_parameters(coordinates, load_parameters)
 
+        # elif isinstance(load_parameters, (PointLoad, LineLoad, SurfaceLoad)):
+        #     # TODO self.__validate_load_coordinates(coordinates)
+        #     pass
+
```

Comment:
can this be removed? is it still relevant?

Reply:
I think is still required, otherwise, now we can add a surface load with 1 point. it should be checked that ne number of coordinates is >2 and for line and moving load the coordinates are >1.